src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureValueController.php

```diff
@@ -48,14 +56,115 @@ public function indexAction(Request $request, FeatureValueFilters $filters): Res
+    }
+
+    /**
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
+     *
+     * @param int $featureId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function createAction(int $featureId, Request $request): Response
+    {
+        $featureValueFormBuilder = $this->get('prestashop.core.form.identifiable_object.builder.feature_value_form_builder');
+        $featureValueFormHandler = $this->get('prestashop.core.form.identifiable_object.handler.feature_value_form_handler');
+
+        try {
+            $featureValueForm = $featureValueFormBuilder->getForm(['feature_id' => $featureId]);
+            $featureValueForm->handleRequest($request);
+            $handlerResult = $featureValueFormHandler->handle($featureValueForm);
+
+            if (null !== $handlerResult->getIdentifiableObjectId()) {
+                $this->addFlash('success', $this->trans('Successful creation', 'Admin.Notifications.Success'));
+
+                if ($request->request->has(self::SAVE_AND_ADD_BUTTON_NAME)) {
+                    return $this->redirectToRoute('admin_feature_values_add', [
+                        'featureId' => $featureId,
+                    ]);
+                }
+
+                return $this->redirectToRoute('admin_features_index');
+            }
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+
+            return $this->redirectToRoute('admin_features_index');
+        }
+
+        return $this->render('@PrestaShop/Admin/Sell/Catalog/Features/FeatureValue/create.html.twig', [
+            'featureId' => $featureId,
+            'featureValueForm' => $featureValueForm->createView(),
+            'layoutTitle' => $this->trans('New Feature Value', 'Admin.Navigation.Menu'),
+        ]);
+    }
+
+    /**
+     * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))")
+     *
+     * @param int $featureValueId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function editAction(int $featureId, int $featureValueId, Request $request): Response
+    {
+        $featureValueFormBuilder = $this->get('prestashop.core.form.identifiable_object.builder.feature_value_form_builder');
+        $featureValueFormHandler = $this->get('prestashop.core.form.identifiable_object.handler.feature_value_form_handler');
+
+        try {
+            $featureValueForm = $featureValueFormBuilder->getFormFor($featureValueId);
+            $featureValueForm->handleRequest($request);
+            $handlerResult = $featureValueFormHandler->handleFor((int) $featureValueId, $featureValueForm);
+
+            if ($handlerResult->isSubmitted() && $handlerResult->isValid()) {
+                $this->addFlash('success', $this->trans('Successful update', 'Admin.Notifications.Success'));
+
+                if ($request->request->has(self::SAVE_AND_ADD_BUTTON_NAME)) {
```

Comment:
I get the principle of having a `Save and add` button in the creation form. For the edition one I'm not sure the UI is that intuitive but why not, QA will point out if it's not a good behaviour I guess